test/owncloud_add_test.cmake

```diff
@@ -6,7 +6,7 @@ function(owncloud_add_test test_class)
     set(OWNCLOUD_TEST_CLASS ${test_class})
     string(TOLOWER "${OWNCLOUD_TEST_CLASS}" OWNCLOUD_TEST_CLASS_LOWERCASE)
 
-    ecm_add_test(test${OWNCLOUD_TEST_CLASS_LOWERCASE}.cpp ${ARGN}
+    ecm_add_test(test${OWNCLOUD_TEST_CLASS_LOWERCASE}.cpp ${PROJECT_SOURCE_DIR}/test/testutils/testutilsloader.cpp ${ARGN}
```

Comment:
Please add a comment that you cannot put this file into a library because that breaks that Qt macro so we don't forget in the future.